src/widgetApp/mainwindow.cpp

```diff
@@ -1,16 +1,311 @@
 #include "mainwindow.h"
 #include "ui_mainwindow.h"
+#include "IPv4parser.h"
+#include "SubnetsCalculatorV4.h"
+#include "IPstructs.h"
+#include <QSpinBox>
+#include <QLabel>
+#include <QtCore>
```

Comment:
Kolejność includów - [clang-tidy](https://llvm.org/docs/CodingStandards.html#include-style)
w naszym projekcie:
1. Main Module Header
2. System #includes
3. Local/Private Headers

```suggestion
#include "mainwindow.h"
#include "ui_mainwindow.h"

#include <QSpinBox>
#include <QLabel>
#include <QtCore>

#include "IPv4parser.h"
#include "SubnetsCalculatorV4.h"
#include "IPstructs.h"
```